### PR TITLE
Feature/189 fix email shipping

### LINKED ISSUE #189: Fix email shipping
There are no emails sent yet on successful extraction.

Estimated duration: ~1d

### PR DESCRIPTION
**Pull request by njordan-hsr**
Closes #189 

Ran tests with `RUN_E2E=true ./test.sh`, all green.

Reviewd by:

* [ ] @das-g 
* [x] @wasabideveloper

### CODE REVIEW

**Comment by wasabideveloper** [comment]
@njordan-hsr You didn't test the email shipping out of the test scope (=real shipping), right?

**Comment by njordan-hsr** [comment]
What do you mean with real shipping? `send_mail` is a Django function, which shouldn't be needed to be tested on our side, so unless we're doing something wrong in our settings, this does work.

This doesn't work 'in reality' since it need the changes from #214 to be able to send mails correctly.

I merged that locally and ran it manually: it sends emails to the console, also using some valid smtp settings, I got a result email, if that is what you're after.

Ran `RUN_E2E=true ./test.sh` after last commit (b43979d), all green.

**Comment by wasabideveloper** [comment]
I don't want to test django `send_mail` but I would like to see the whole setup working (e.g. routing the email out of the docker forest...). But it's ok. I will merge it an we will test it after.
:heavy_check_mark:

File: osmaxx-py/excerptconverter/converter_helper.py [hunk 1]
```diff
@@ -70,14 +72,12 @@ def inform_user(self, message_type, message_text, email=True):
...
-                pass
-                # FIXME: fix email sending
-                # send_mail(
-                #     '[OSMAXX] '+message_text,
-                #     message_text,
-                #     'no-reply@osmaxx.hsr.ch',
-                #     [self.user.email]
-                # )
+                mail.send_mail(
+                    '[OSMAXX] '+message_text,
```
Comment (das-g) [inline osmaxx-py/excerptconverter/converter_helper.py]: Isn't the hardcoded prefix here redundant to the `EMAIL_SUBJECT_PREFIX` environment variable?

**Pull request merged**